core/store/src/trie/trie_storage.rs

```diff
@@ -231,21 +252,36 @@ impl TrieCachingStorage {
 
 impl TrieStorage for TrieCachingStorage {
     fn retrieve_raw_bytes(&self, hash: &CryptoHash) -> Result<Arc<[u8]>, StorageError> {
+        let shard_id_str = format!("{}", self.shard_uid.shard_id);
+        let is_view_str = format!("{}", self.is_view as u8);
+        let labels: [&str; 2] = [&shard_id_str, &is_view_str];
+        {
+            metrics::CHUNK_CACHE_SIZE
+                .with_label_values(&labels)
+                .set(self.chunk_cache.borrow().len() as i64);
+            metrics::SHARD_CACHE_SIZE
+                .with_label_values(&labels)
+                .set(self.shard_cache.0.lock().expect(POISONED_LOCK_ERR).len() as i64);
+        }
```

Comment:
Let's separate it to `fn update_cache_size_metrics()` for `TrieCachingStorage` and drop extra curly braces

Reply:
Sorry, I forgot that we use `labels` in this method below. Then the original code makes more sense